js/init.js

```diff
@@ -1,4 +1,4 @@
-const jsScripts = [ 'jquery.bind-first', 'dompurify', 'keymaster', 'glotdict-locales', 'glotdict-functions', 'glotdict-settings', 'glotdict-hotkey', 'glotdict-validation', 'glotdict-column', 'glotdict-meta', 'glotdict-bulk', 'glotdict-notices', 'glotdict-consistency', 'glotdict' ];
+const jsScripts = [ 'jquery.bind-first', 'dompurify', 'keymaster', 'glotdict-locales', 'glotdict-functions', 'glotdict-settings', 'glotdict-hotkey', 'glotdict-validation', 'glotdict-column', 'glotdict-meta', 'glotdict-bulk', 'glotdict-notices', 'glotdict', 'glotdict-consistency' ];
```

Comment:
maybe consistency is the case that is loaded before then glotdict general :-)

Reply:
In order to use `gd_user object` that is declared in glotdict.js we need to load glotdict.js script before.
That was the reason behind.

Reply:
Ok so I am wondering if it is the case to have a file with all those definitions to run as first and later the rest.

Reply:
yes it is the best way to achieve it :-)